license-header.txt

```diff
@@ -0,0 +1,21 @@
+/*
+ * This file is part of INSPIRE.
```

Comment:
Actually I copied it from
https://github.com/inspirehep/inspirehep-search-js/blob/master/src/inspirehep-search-js/filters/abstract.js#L2

will update to `record-editor` soon, anyway.